### ISSUE TITLE
RedisJSON for Redis Cluster?

### ISSUE THREAD

**Issue by radoye** [RedisJSON/redisjson-py#39]
Currently Client [is a](https://github.com/RedisJSON/redisjson-py/blob/master/rejson/client.py#L27) `StrictRedis` client. This does not seem to work with the Redis Cluster setup (MOVED error due to redirection that is not followed?). 

Does it make sense to pass a redis/rediscluster object to Client to make the connector pluggable?

**Comment by gkorland** [comment]
Yes, it does you're right, it is on our list. If you're willing to make a PR that will be great.
